packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -246,6 +248,56 @@ describe('build-schema', () => {
       expect(schema.definitions).to.be.undefined();
     });
 
+    it('omits title for relation model definition', () => {
+      @model()
+      class Child extends Entity {
+        @property({
+          type: 'string',
+        })
+        name?: string;
+      }
+      @model()
+      class Parent extends Entity {
+        @hasMany(() => Child)
+        children: Child[];
+
+        @property({
+          type: 'string',
+        })
+        benchmarkId?: string;
+
+        @property({
+          type: 'string',
+        })
+        color?: string;
+
+        constructor(data?: Partial<Parent>) {
+          super(data);
+        }
+      }
+      const schema = modelToJsonSchema(Parent, {
+        title: 'ParentWithItsChildren',
+        includeRelations: true,
+      });
+      expect(schema.properties).to.containEql({
+        children: {
+          type: 'array',
+          items: {$ref: '#/definitions/ChildWithRelations'},
+        },
+        benchmarkId: {type: 'string'},
+        color: {type: 'string'},
+      });
+      // No circular references in definitions
+      expect(schema.definitions).to.containEql({
+        ChildWithRelations: {
+          title: 'ChildWithRelations',
+          description: '(Schema options: { includeRelations: true })',
+          properties: {name: {type: 'string'}},
+          additionalProperties: false,
+        },
```

Comment:
LGTM. Just have a question. If `Parent` has many `Parent`, would the title here be `PatentWIthRelations`?

Reply:
@agnes512 For self relations, the same schema is cached so that [line 419](https://github.com/strongloop/loopback-next/blob/344784be9d9b7624952db1f17473d5308ba2ae70/packages/repository-json-schema/src/build-schema.ts#L419) would return.

We also have a test case for it just above the new ones I added in this PR :)